### PR TITLE
Release/1.3.0

### PR DESCRIPTION
**Pull request by lindenmckenzie**
### What

Added table support
Add aggregated pages /data endpoint migration guide

### How to review

Check reads ok, nothing unnecessary in release. 

### Who can review

Not me.

### CODE REVIEW

File: static/retirement/aggregatedsearch/index.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+## Background
+
+In line with our [approach to the retirement of API endpoints](../), it has been decided that ONS will retire the data endpoints for our aggregated search pages due to the following reasons:
+
+- out of date technology
+- built as a proof of concept
+- strategic technology direction is elsewhere
+- cost of upkeep is too high with similar services available
+
+The UI for these pages has already been moved to using our Search API and it is the intention of ONS to separate data and UI endpoints by domain in future. Data in JSON format will be accessible via our API found at <https://api.beta.ons.gov.uk/v1/search>. You can also browse the [full API Swagger specification](/search/search).
```
Comment (franmoore05) [inline static/retirement/aggregatedsearch/index.md:14]: Should "using" be "use" here?

Comment (lindenmckenzie) [reply static/retirement/aggregatedsearch/index.md:14]: yeah probably :D

Comment (lindenmckenzie) [reply static/retirement/aggregatedsearch/index.md:14]: https://github.com/ONSdigital/dp-developer-site/pull/92

**Review by franmoore05 — DISMISSED** [review]
Happy to approve based on PR #92

**Pull request merged**